.github/e2e-tests.yml

```diff
@@ -939,6 +922,24 @@ runner-test-matrix:
 
   # END: Other tests
 
+  # START: CCIPv1.6 tests
+
+  - id: smoke/ccip_test.go:*
+    path: integration-tests/smoke/ccip_test.go
+    test_env_type: docker
+    runs_on: ubuntu-latest
+    triggers:
+      - PR E2E CCIP Tests
+      - Merge Queue E2E CCIP Tests
```

Comment:
I changed triggers to "CCIP". This means the tests will only run when CCIP code is changed.